references/classification/utils.py

```diff
@@ -161,6 +161,25 @@ def log_every(self, iterable, print_freq, header=None):
         print('{} Total time: {}'.format(header, total_time_str))
 
 
+class ExponentialMovingAverage(torch.optim.swa_utils.AveragedModel):
+    """Maintains moving averages of model parameters using an
```

Comment:
Doc nits that don't really matter since we don't render the docs, but good to fix if we ever move this in the public API. Also good to know for future PRs:

- the first sentence should be in one line according to pep8. When this isn't respected, sphinx can create rendering issue (typically when displaying the summary of a function / class)
- since sphinx relies on rst format (not markdown like in github), the single backticks should be double backticks.
- I think there's a missing `_` at the end of the link below, although I haven't checked

Reply:
Thanks, updated them.